pkg/kubectl/describe.go

```diff
@@ -1779,10 +1779,18 @@ func (d *ServiceAccountDescriber) Describe(namespace, name string, describerSett
 		}
 	}
 
-	return describeServiceAccount(serviceAccount, tokens)
+	existingPullSecrets := map[string]api.Secret{}
+	pullSecrets, err := d.Core().Secrets(namespace).List(api.ListOptions{})
+	if err == nil {
```

Comment:
Please take a look at this [part](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go#L3242)